client/components/LearnerViewDirective.js

```diff
@@ -24,24 +24,41 @@ tie.directive('learnerView', [function() {
       <h3>Exercise: {{title}}</h3>
 
       <div class="tie-learner-view-left-column">
+        <div class="tie-old-instructions">
```

Comment:
I'm not sure what the difference is between old-instructions and instructions. Could you clarify?